docs/modules/ROOT/pages/flow.adoc

```diff
@@ -0,0 +1,77 @@
+= Deploy Flow
+
+:description: In this tutorial, you'll deploy a Flow application using Hazelcast Platform Operator for Kubernetes.
+
+{description}
+
+See xref:flow:introduction:index.adoc[Flow] for more detailed information.
+
+== Prerequisites
+
+- Flow requires the Management Center application to run and be configured properly. The Management Center application should be configured to use FLow.
+
+[source,yaml]
+----
+...
+image: docker.io/hazelcast/management-center-flow <1>
+...
+hazelcastClusters:
+- address: flow <2>
+  name: flow <3>
+jvm:
+  args:
+  - "-Dhazelcast.mc.flow.addresses=http://233.252.0.158/" <4>
```

Comment:
Not sure if you commonly use this elsewhere, but this ip in the examples looks slightly strange to me, it being ASM reserved makes it look super specific, but I presume it's just a placeholder, could perhaps replace with a placeholder FQDN instead?

Reply:
Well, I specifically used the IP range reserved for documentation purposes as a placeholder. In the description, I have explained that it should be publicly available one 😄